docs/source/examples/shapes_images_positions.rst

```diff
@@ -43,4 +68,146 @@ Shapes, Images and Positions
             self.add(img_mobject)
 
 
-Note: Here can come the UnitDot Example.
\ No newline at end of file
+.. manim:: ArcShapeIris
+    :save_last_frame:
+    
+    class ArcShapeIris(Scene):
+        def construct(self):
+            colors = [DARK_BLUE , DARK_BROWN, BLUE_E, BLUE_D, BLUE_A, TEAL_B, GREEN_B, YELLOW_E]
+            radius = [1+rad*0.1 for rad in range(len(colors))]
+            
+            circles_group=VGroup()
+            
+            # zip(radius,color) makes the iterator [(radius[i],color[i]) for i in range(radius)]
+            circles_group.add( *[Circle(radius=rad,stroke_width=10, color = col)
+                         for rad,col in zip(radius,colors)])
+            
+            self.add(circles_group)
```

Comment:
This is a minor variation to what black would do: I think linebreaking after every entry in the `colors` list does not really make the example more readable.

```suggestion
    class ArcShapeIris(Scene):
        def construct(self):
            colors = [DARK_BLUE, DARK_BROWN, BLUE_E, BLUE_D, BLUE_A, TEAL_B, GREEN_B, YELLOW_E]
            radius = [1 + rad * 0.1 for rad in range(len(colors))]

            circles_group = VGroup()

            # zip(radius, color) makes the iterator [(radius[i], color[i]) for i in range(radius)]
            circles_group.add(*[Circle(radius=rad, stroke_width=10, color=col) 
                                for rad, col in zip(radius, colors)])
            self.add(circles_group)
```